content/en/docs/concepts/storage/volumes.md

```diff
@@ -335,12 +335,16 @@ Some uses for an `emptyDir` are:
 * holding files that a content-manager container fetches while a webserver
   container serves the data
 
-Depending on your environment, `emptyDir` volumes are stored on whatever medium that backs the
-node such as disk or SSD, or network storage. However, if you set the `emptyDir.medium` field
-to `"Memory"`, Kubernetes mounts a tmpfs (RAM-backed filesystem) for you instead.
-While tmpfs is very fast, be aware that unlike disks, tmpfs is cleared on
-node reboot and any files you write count against your container's
-memory limit.
+Depending on your environment, `emptyDir` volumes are stored on whatever medium
+that backs the node such as disk or SSD, or network storage. A size limit can be
+specified, which counts against the [node ephemeral
+storage](docs/concepts/configuration/manage-resources-containers/#setting-requests-and-limits-for-local-ephemeral-storage)
+allocation.
```

Comment:
"which counts against the allocation". Does it mean certain amount of storage is allocated? If so, I feel it is different from the meaning the sizeLimit. It tries to limit how much it can be consumed..